scope.go

```diff
@@ -478,9 +478,12 @@ func (s *scope) Snapshot() Snapshot {
 }
 
 func (s *scope) Close() error {
+	// prevent any more report loops via tick before forcing report
+	close(s.status.quit)
```

Comment:
We should probably add a check to ensure that we don't panic due to closing chan twice by calling close twice.